_data/roles.yml

```diff
@@ -0,0 +1,113 @@
+- name: Registry Metadata Steward
+  open: true
+  commitment: 2-4 hours per month
+  requirements:
+    - Love for metadata
+    - basic python
+    - community mindset
+  description: |
+    You are responsible for shepherding our registry
+    metadata according to our principles and SOPs, merge changes and generally
+    protect and guard them. This involves also facilitating the implementation of QC
+    checks.
+- name: OBO NOR Dashboard maintainer
+  open: true
+  commitment: 2-6 hours per month
```

Comment:
I would like to propose an element `responsibilities` - I will fill it in. It will detail what needs to be done for each roll exactly. So when yo assume a role you know exactly the 10 things that need to be done.

Reply:
I added a line for this to each role. Thanks for filling it in @matentzn